tools/kontaktdaten.py

```diff
@@ -91,6 +169,8 @@ def validate_kontakt(kontakt):
             if key in ["anrede", "vorname", "nachname", "strasse", "ort"]:
                 if not isinstance(value, str):
                     raise ValidationError("Muss eine Zeichenkette sein")
+                if value.strip() == "":
+                    raise ValidationError(f"Fehlende Daten bei {key}")
```

Comment:
Key angeben ist hier nicht nötig, da das beim abfangen und reraisen der Exception gemacht wird.

```suggestion
                    raise ValidationError(f"Darf nicht leer sein")
```